eng/pipelines/docindex.yml

```diff
@@ -23,7 +23,9 @@ jobs:
         displayName: Download and Extract DocFX
 
       - pwsh: |
-          $(Build.SourcesDirectory)/eng/docgeneration/Generate-DocIndex.ps1 -RepoRoot $(Build.SourcesDirectory) -DocGenDir "$(Build.SourcesDirectory)/eng/docgeneration" -verbose
+          $(Build.SourcesDirectory)/eng/docgeneration/Generate-DocIndex.ps1 
+          -Docfx $(Resolve-Path "$(Build.SourcesDirectory)/docfx/docfx.exe")[0]
```

Comment:
This Resolve-Path looks kind of odd and could be error prone. Instead I suggest setting a path as an output variable in the download and install step before this.

Reply:
Sure. This copies from copy-docs-to-blobstorage scripts. Will have a variable for the path